src/resources/benefitCompany.ts

```diff
@@ -0,0 +1,100 @@
+import { ResourceIF } from '@/interfaces'
+import { NameRequestTypes, RouteNames } from '@/enums'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+export const BenefitCompanyResource: ResourceIF = {
+  entityType: CorpTypeCd.BENEFIT_COMPANY,
+  displayName: GetCorpFullDescription(CorpTypeCd.BENEFIT_COMPANY),
+  title: 'Benefit Company Statement',
+  description: `This company is a benefit company and, as such, has purposes that include conducting its business
+        in a responsible and sustainable manner and promoting one or more public benefits.`,
+  statement: null,
+  nameRequestType: NameRequestTypes.BC,
+  steps: [
+    {
+      id: 'step-1-btn',
+      step: 1,
+      icon: 'mdi-domain',
+      text: 'Define Your\nCompany',
+      to: RouteNames.DEFINE_COMPANY,
+      component: 'define-company'
```

Comment:
I know these are strings. Where do we get these? From an import? Should we have an enum? 🤷

Reply:
Probably should! I don't mind doing it now.

Reply:
Well, where did these strings come from?

Reply:
The route names, which can be re used here.

Reply:
Updated!